.devcontainer/xauth_setup.sh

```
@@ -2,7 +2,11 @@
 
 # Script to create a Xauthority file for the docker container
 XAUTH=$HOME/.Xauthority
+touch $XAUTH
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)